src/sentry/runner/commands/devserver.py

```diff
@@ -232,9 +237,9 @@ def devserver(
     # A better log-format for local dev when running through honcho,
     # but if there aren't any other daemons, we don't want to override.
     if daemons:
-        uwsgi_overrides["log-format"] = '"%(method) %(status) %(uri) %(proto)" %(size)'
+        uwsgi_overrides["log-format"] = "%(method) %(status) %(uri) %(proto) %(size)"
     else:
-        uwsgi_overrides["log-format"] = '[%(ltime)] "%(method) %(status) %(uri) %(proto)" %(size)'
+        uwsgi_overrides["log-format"] = "[%(ltime)] %(method) %(status) %(uri) %(proto) %(size)"
```

Comment:
maybe yall like having the quotes. I didn't care either way.

Reply:
Does anyone use protocol/size?

Reply:
Yeah I don't know. Maybe? I definitely do not.

Reply:
MAYBE size could be useful like, 1 in a 1000 times

Reply:
They are useful for debugging weird situations

Reply:
It was intended to mirror apache log format. The chunk in quotes is the "request line" of HTTP.